lib/td/command/job.rb

```diff
@@ -493,7 +496,7 @@ def render_result(job, limit, format=nil, render_opts={})
       job.result_each_with_compr_size {|row, compr_size|
         indicator.update(compr_size)
         rows << row.map {|v|
-          dump_column_safe_utf8(v)
+          dump_column_safe_utf8(v, render_opts[:null_expr])
```

Comment:
In this scope,  `render_opts[:null_expr]` is used only once.
In the other method `#write_result`,  it is used twice in the case of csv and tsv, but not in json.
I think it is better to leave the code as is because of the clarity of case difference.